rotkehlchen/exchanges/ftx.py

```diff
@@ -103,6 +105,11 @@ def __init__(
         self.base_uri = 'https://ftx.com'
         self.msg_aggregator = msg_aggregator
         self.session.headers.update({'FTX-KEY': self.api_key})
+        self.subaccount = None
+        subaccount = self.db.get_ftx_subaccount(self.name)
+        if subaccount is not None:
+            self.session.headers.update({'FTX-SUBACCOUNT': quote(subaccount)})
```

Comment:
why use quote? Here? Sanitization I guess but against what?

Reply:
I used it here because FTX used them in their example. I didn't remove it because I believe could be something also in the FTX server when doing the validation https://github.com/ftexchange/ftx/blob/master/rest/client.py#L46